### PR TITLE
Windows build for mpc

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,48 @@
 {% set version = "1.0.3" %}
+{% set commit = "58fa4d15fff10f26c490518bc0da2b69732e22ee" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:2]: Could we ping him about getting some tags?

What is special/what do we know about this commit?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,33 +1,48 @@
...
+  fn: {{ commit }}.zip                                                      # [win]
+  url: https://github.com/BrianGladman/mpc/archive/{{ commit }}.zip         # [win]
+  md5: a3ccbbb17770110427a983638824f31b                                     # [win]
 
 build:
   number: 1
-  skip: true  # [win]
+  skip: true          # [win and not py35]
+  features:
+    - vc14            # [win and py35]
```
Comment (jakirkham) [inline recipe/meta.yaml:20]: Guessing it doesn't work with older VC versions. Correct?

File: recipe/bld.bat [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+)
+
+REM copy headers
+xcopy %LIBRARY_INC%\*.h mpir\lib\%PLATFORM%\Release\ /E
+xcopy %LIBRARY_INC%\*.h mpir\dll\%PLATFORM%\Release\ /E
+
+cd %mpc_root%\build.vc14
+
+msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release /p:PostBuildEvent="" mpc_lib\mpc_lib.vcxproj
+msbuild.exe /p:Platform=%PLATFORM% /p:Configuration=Release /p:PostBuildEvent="" mpc_dll\mpc_dll.vcxproj
```
Comment (jakirkham) [inline recipe/bld.bat:30]: Is there some sort of test suite that we can run here?

**Comment by jakirkham** [comment]
Could you please confirm how you generated `appveyor.yaml`?

File: recipe/meta.yaml (hunk 2 above)
Comment (isuruf) [inline recipe/meta.yaml:20]: Correct

**Comment by isuruf** [comment]
`conda-smithy regenerate`, but it seems I generated it when `python` was used, not `vc 14`. Now when I regenerate appveyor.yml is disabled.

**Comment by jakirkham** [comment]
Yeah, we haven't added logic for `vc` yet. Sounds fine. Don't worry about disabled AppVeyor part.

File: recipe/bld.bat (hunk 3 above)
Comment (isuruf) [inline recipe/bld.bat:30]: Yes, but the script is an interactive python script. I'll have to write it myself for command line use

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [inline recipe/meta.yaml:2]: Nothing special, this is a commit including the MSVC build files that is in between 1.0.3 and the development version.

Comment (isuruf) [inline recipe/meta.yaml:2]: I think the developer is using MPC original git repo and updates his repo from time to time with the MSVC build files rebased

**Comment by isuruf** [comment]
Anything to be done here?

**Comment by jakirkham** [comment]
Some sort of testing for Windows would be nice. Perhaps a simple program that exercises some functionality from `mpc`.

**Comment by isuruf** [comment]
Tests pass now

**Comment by jakirkham** [comment]
@msarahan, could you please take a look at this and see if this looks ok to you?

**Comment by isuruf** [comment]
@msarahan, ping

**Comment by isuruf** [comment]
pinging again

**Comment by jakirkham** [comment]
He's been pretty busy trying to fix `conda` and `conda-build` bugs. Let's give him a little more time.

**Comment by jakirkham** [comment]
In the mean time, @patricksnape could you please help review this when you have a chance?

**Comment by msarahan** [comment]
All looks sane to me.  Why is there a custom appveyor.yaml, though?

**Comment by jakirkham** [comment]
What do you mean by "custom"? I believe this was generated with `conda smithy regenerate`.

**Comment by msarahan** [comment]
I see, sorry.  I didn't understand why the file was part of this PR at all.

**Comment by jakirkham** [comment]
I tried to regenerate locally and basically get the same result. Think that @isuruf may have a slightly older version of `conda smithy` though.

However, there definitely seems to be something funny happening in `conda-smithy` w.r.t. the `skip` portion. Seems it decides to skip all Windows builds instead of all Windows builds that are not Python 3.5. 😕 @pelson, do you have any insight on why this is happening?

**Comment by jakirkham** [comment]
NVM we don't list `python` and `conda-smithy` doesn't know about `vc` yet. Sorry for the noise.

**Comment by jakirkham** [comment]
I've added a PR ( https://github.com/isuruf/mpc-feedstock/pull/1 ) against your PR, @isuruf, so that we can fix the re-rendering issue.

**Comment by jakirkham** [comment]
> I see, sorry. I didn't understand why the file was part of this PR at all.

Yeah, so normally this is less pronounced as it is results in just moving `disabled_appveyor.yml` to `appveyor.yml`. Potentially some differences appear. However, we didn't have `disabled_appveyor.yml` either so it came across as a totally new file. I'm not totally sure why it didn't already exist though.

**Comment by jakirkham** [comment]
Does this still look good to you, @msarahan? If so, I'm happy to see this merged once it passes.

**Comment by msarahan** [comment]
LGTM, OK to merge when green.

**Comment by jakirkham** [comment]
The queues are really backed up so it might not finish until late. If it passes, please feel free to self-merge so that you can get started on other things, @isuruf.

**Comment by isuruf** [comment]
Thanks for the reviews

**Pull request merged**